csharp/src/Ice/IPEndpoint.cs

```diff
@@ -356,11 +351,6 @@ private protected IPEndpoint(InputStream istr, Communicator communicator, Protoc
 
         private protected abstract IConnector CreateConnector(EndPoint addr, INetworkProxy? proxy);
 
-        // TODO: rename to Clone and make parameters optional?
-        private protected abstract IPEndpoint CreateIPEndpoint(
-            string host,
-            ushort port,
-            bool compressionFlag,
-            TimeSpan timeout);
+        private protected abstract IPEndpoint CloneWithHostAndPort(string host, ushort port);
```

Comment:
Should we call this method simply Clone?

Reply:
fixed